### ISSUE TITLE
[Request]: Bypass censorship in chat

### ISSUE THREAD

**Issue by gfhghfghfhgfyututyuykyghgn** [YimMenu/YimMenu#3208]
### Problem

Censorship in chat is very annoying. Or can it be disabled in the settings?

### Solution

Is it possible to bypass censorship in chat, for example, display player messages without censorship? Or at least overcome censorship yourself and call someone a nіgger.

### Reason

I want to call someone a nіgger

**Comment by gir489returns** [comment]
This used to be possible a long time ago, since the censorship was handled by the client before it emitter. Calling someone the niggery nigger N word was exactly the usecase I wanted it for. When I posted how to do it to UC, I posted the "GENTLEMEN! I HAVE COME TO SAY THE N WORD!" video.

**Comment by Rxann** [comment]
> You can replace characters with homographs and you wouldn't be censored. That's how the P2Cs implement their bypass after the patch

Yes, we could do something with a map of homographs and their translation if we wanted do this feature.

> Also some people should just grow up

I agree, if it is censored by Rockstar, maybe it just shouldn't be said.
